src/main/java/com/parallels/desktopcloud/ParallelsDesktopConnectorSlaveComputer.java

```diff
@@ -411,4 +414,34 @@ public boolean hasPermission(Permission permission)
 			return false;
 		return super.hasPermission(permission);
 	}
+
+	@Override
+	public void setTemporarilyOffline(boolean temporarilyOffline, OfflineCause cause)
+	{
+		try
+		{
+			Jenkins jenkins = Jenkins.getInstance();
+			for (ParallelsDesktopVM vm : this.getNode().getOwner().getVms())
+			{
+				Node slaveNode = jenkins.getNode(vm.getSlaveName());
+				if (slaveNode == null)
+				{
+					continue;
+				}
+				Computer slaveComputer = slaveNode.toComputer();
+				if (!slaveComputer.isOffline())
```

Comment:
So you're unable to bring offline node to online. Seems you may just omit `isOffline` condition and call `setTemporarilyOffline` directly.